components/sp-solutions/org.wso2.analytics.solutions.sp.twitter.analytics/widgets/TopSentiment/src/TopSentiment.jsx

```diff
@@ -14,35 +14,49 @@
  * limitations under the License.
  */
 
-import React, {Component} from 'react';
+import React from 'react';
 import Widget from '@wso2-dashboards/widget';
+import {MuiThemeProvider} from 'material-ui/styles';
+import {IconButton} from 'material-ui';
```

Comment:
use import IconButton from '@material-ui/core/IconButton'; to avoid entire module gettting packed and effecting the bundle size.

Reply:
Fixed with #303